pyqt-apps/siriushla/as_di_dccts/settings.py

```diff
@@ -1,14 +1,16 @@
 """DCCT settings module."""
 
+import epics
```

Comment:
@anacso17 , @fernandohds564 , should we use siriuspy.epics instead ?
we had been doing this so that moving to another python EPICS client lib, if necessary, would be easier...

Reply:
I think it is a good practice, @xresende. Sometimes I don't pay attention to this, but I think it is a good idea to do it and to point it out in our reviews!

Reply:
done!